INSTRUMENTS/Stepper_Motor/STEPPERDRIVER_TIC_KNOB.py

```diff
@@ -0,0 +1,37 @@
+from flojoy import flojoy, DataContainer
+from ticlib import TicUSB
+from time import sleep
+
+
+@flojoy
+def STEPPERDRIVER_TIC_KNOB(dc, params):
+    tic = (
+        TicUSB()
+    )  # Declaration of the stepper driver (You can add serial number to specify the driver)
+
+    tic.set_current_limit(
+        30
```

Comment:
Never use "magic numbers"

All numbers should be settable through `params` if possible, then set to sensible defaults (like 30 in this case) if they are not provided

If you have a constant that does not make sense to make settable as a param with a sensible default, then you shoudl always define it as a constant variable at the top of the function

for example
```
CURRENT_LIMIT = params['current_limit'] if current_limit in params else 30
```

Reply:
done, I put it in params with a default value of 30, it makes sense to put it in params because the value will change depending on the size of the motor and the TIC driver model